updater/src/Updater/Generate/Template.purs

```diff
@@ -154,75 +146,90 @@ runTemplate opts (Template { from, to }) = do
 
   FS.Extra.writeTextFile to (replaceVariables opts.variables templateContents)
 
+-- | Keep JS only templates when using JS.
+filterByType :: Boolean -> Array TemplateSource -> Array TemplateSource
```

Comment:
In general I prefer not to use boolean arguments because then you have to read the source to understand what it means. What do you think of either adding a dedicated type to represent whether we're using JS or not, or switching these functions to take records instead?

For example, this is clearer to me:

```purs
filterByType :: { usesJS :: Boolean, templates :: Array TemplateSource } -> Array TemplateSource
```

despite being a bit more verbose.

Reply:
Yes, the record makes it clearer!